dvc/stage/run.py

```diff
@@ -110,11 +114,8 @@ def run_stage(stage, dry=False, force=False, checkpoint_func=None, **kwargs):
 
     callback_str = "callback " if stage.is_callback else ""
     logger.info(
-        "Running %s" "stage '%s' with command:",
-        callback_str,
-        stage.addressing,
+        "Running %s" "stage '%s':", callback_str, stage.addressing,
```

Comment:
Hi! Why no longer "with command" ? Is there a full example of what the output of run will be for both a single command stage and a multi-line? Thanks!

Reply:
And this code is also used by repro right? (Or does it need similar update there?)

Reply:
Actually I have moved the "command" to a new logging line in `cmd_run` (line 82) so as to be able to log the current command. This code is also used by repro indeed.

Reply:
But it doesn't seem to output "with command(s):" anymore (per https://github.com/iterative/dvc/pull/4934#discussion_r533048696 below) — that's what I'm referring to. It would be great if you can provide a multi-cmd dvc.yaml sample + its repro output, at least for the docs review 🙏. That will make things easier 🙂

Reply:
It is not clear to me where I should add this example

Reply:
Resolving here because I have added tests with multiple commands; in the dvc.doc I will add some examples